easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,59 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'ConfigureMake'
+
+name = 'GetFEM'
+version = '5.4.1'
+local_pyver = '3.7.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://getfem.org'
```

Comment:
@jordiblasco do you know what's going on with the homepage? I tried the `https` version of that address, and it had a strange result.

Reply:
@jordiblasco we still need to deal with the homepage error - can it be `https` or not?